website/src/pages/create/summarize_story.tsx

```diff
@@ -5,7 +5,7 @@ import useSWRImmutable from "swr/immutable";
 import useSWRMutation from "swr/mutation";
 
 import { Footer } from "src/components/Footer";
-import { Header } from "src/components/Header";
+import { Header } from "@/components/Header/Header";
```

Comment:
can we replace `@` with `src`?
I know vscode automatically added it....

Reply:
I removed the line entirely while merging..